### DISCUSSION TITLE
Upgrade causing 552 error with uploads (Version 8.5.8 (39606))

### DISCUSSION THREAD

**Discussion by Blue-Sailing** [iterate-ch/cyberduck#14501] [Q&A]
Hey guys, I upgraded this morning my Cyberduck and now all of my uploads are receiving error 552. My file manager in cpanel uploads fine without error and all disk usage is normal. In essence, there is plenty of disk room but Cyberduck is failing. Please advise...Thanks!

My System: macOS Monterey, Version 12.6.4, Intel

**Comment by dkocher** [comment]
The code `552` suggests that you have run out of available disk space.

**Reply by Blue-Sailing** [reply]
@dkocher please read my issue more closely, I understand the error. My server has plenty of disk space, only using 2gb of 1tb. Also do not encounter problem with other other uploads not using cyberduck. I didn't have the issue with Cyberduck either, until after the upgrade.

FYI: Today, after rebooting and clearing my mac log files, I added another 1gb of data to my server on purpose through the cpanel. Opened up Cyberduck and tried to upload the same 87kb file from yesterday. It uploaded without a problem!??? So now it is working fine.
